test/common/testlib.py

```diff
@@ -854,6 +857,14 @@ def skipImage(reason, *args):
         return unittest.skip("{0}: {1}".format(testvm.DEFAULT_IMAGE, reason))
     return lambda func: func
 
+def skipGeneric(*args):
+    packages_env = os.environ.get(TEST_SKIP_ENV_VARIABLE, "").split(" ")
```

Comment:
Please use `.split()`, which gives the correct result for empty strings. I. e. `"".split(" ") == [""]`, while `"".split() == []`.